index.html

```diff
@@ -7,9 +7,1023 @@
+          <p>
+            <span class="number">#2: </span>
+            <span class="name">THE KEYS</span><span style="color: white"> (15.34%)</span></p>
+         </div>
+        </div>
+        <iframe class="announcer" loading="lazy" allowfullscreen src="https://streamable.com/e/99ytbg"></iframe>
+      </div>
+      <div class="img_award award page" id="emote">
+        <h1 class="page_title">The WEOW Award for Best New Emote of the Year</h1>
+        <div class="nominee_list">
+         <div class="nominee">
+          <img src="Media/Emote/EVIL.webp" loading="lazy">
+          <p>
+            <span class="number">#10: </span>
+            <span class="name">EVIL</span><span style="color: white"> (2.54%)</span></p>
+         </div>
+         <div class="nominee">
+          <img src="Media/Emote/hesRight.webp" loading="lazy">
+          <p>
+            <span class="number">#9: </span>
+            <span class="name">hesRight</span><span style="color: white"> (3.36%)</span></p>
+         </div>
+         <div class="nominee">
+          <img src="Media/Emote/VeryPog.webp" loading="lazy">
+          <p>
+            <span class="number">#8: </span>
+            <span class="name">VeryPog</span><span style="color: white"> (3.83%)</span></p>
+         </div>
+         <div class="nominee">
+          <img src="Media/Emote/SWEEPING.gif" loading="lazy">
+          <p>
+            <span class="number">#7: </span>
+            <span class="name">SWEEPING</span><span style="color: white"> (3.86%)</span></p>
+         </div>
+         <div class="nominee">
+          <img src="Media/Emote/dinkDonk.webp" loading="lazy">
+          <p>
+            <span class="number">#6: </span>
+            <span class="name">dinkDonk</span><span style="color: white"> (4.27%)</span></p>
+         </div>
+         <div class="nominee">
+          <img src="Media/Emote/TINYING.gif" loading="lazy">
+          <p>
+            <span class="number">#5: </span>
+            <span class="name">TINYING</span><span style="color: white"> (5.93%)</span></p>
+         </div>
+         <div class="nominee">
+          <img src="Media/Emote/temmieDank.webp" loading="lazy">
+          <p>
+            <span class="number">#4: </span>
+            <span class="name">temmieDank</span><span style="color: white"> (6.21%)</span></p>
+         </div>
+         <div class="nominee">
+          <img src="Media/Emote/FISHIES.gif" loading="lazy">
+          <p>
+            <span class="number">#3: </span>
+            <span class="name">FISHIES</span><span style="color: white"> (7.28%)</span></p>
+          </div>
+         <div class="nominee">
+          <img src="Media/Emote/Normgasm.webp" loading="lazy">
+          <p>
+            <span class="number">#2: </span>
+            <span class="name">NormGasm</span><span style="color: white"> (21.46%)</span></p>
+         </div>
+         <div class="nominee">
+          <img src="Media/Emote/CINEMA.webp" loading="lazy">

```

Comment:
a few emotes don't seem to load for me, seems like CINEMA, for example, is a jpg in the files, but here u reference a webp

Reply:
ty for letting me know, emotes are replaced now, idk how to fix the streamable thing :( and it's 3am so I'll maybe just leave it like that (most of the clips aren't necessarily spoiled anyway)

also I have just reviewed dgger of the year and after throwing out all of the fraudulent ballots, it appears you have won with 100% of the vote

Reply:
yeah no worries, the streamable thing is a minor nitpick anyway

and pog, finally, after so many years FeelsStrongMan